test/integration/odoWrapper.test.ts

```diff
@@ -100,12 +101,12 @@ suite('./odo/odoWrapper.ts', function () {
         });
 
         test('analyze()', async function () {
```

Comment:
@msivasubramaniaan The `test('analyze()', async function () {` test case should probably also be moved out from `test/integration/odoWrapper.test.ts` into a separate test, shouldn't it?